src/sass/partials/_footer.scss

```diff
@@ -0,0 +1,48 @@
+@use "./variables" as *;
```

Comment:
the scroll bar shows up on your card description & meaning even if there isn't info to scroll but the UX feels unintuitive because it looks like you should be able to scroll. Maybe you can update that box to grow/shrink with content instead?

Reply:
This section was a big struggle w/ varying amounts of data being returned. Will def. revisit. Thx for the heads up